gridfs_bucket.go

```diff
@@ -137,7 +117,7 @@ func (b *gridfsBucket) Writer(ctx context.Context, name string) (io.WriteCloser,
 		"key":           name,
 	})
 
-	grid, err := b.bucket(ctx)
+	grid, err := b.bucket()
```

Comment:
This is a syncronous func, below when we call `OpenUploadStream` is the async/ctx part